otelcolbuilder/cmd/configprovider.go

```diff
@@ -38,8 +38,6 @@ import (
 func UseCustomConfigProvider(params *otelcol.CollectorSettings) error {
 	// feature flags, which are enabled by default in our distro
 	err := featuregate.GetRegistry().Apply(map[string]bool{
-		"receiver.apache.emitServerNameAsResourceAttribute":            true,
-		"receiver.apache.emitPortAsResourceAttribute":                  true,
 		"receiver.elasticsearch.emitClusterHealthDetailedShardMetrics": true,
 		"receiver.elasticsearch.emitAllIndexOperationMetrics":          true,
 	})
```

Comment:
The feature gates for elasticsearch are in beta now (enabled by default), so technically we could remove this, but I'm not sure if in this PR.

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/issues/896